hw/xbox/xbox.c

```diff
@@ -227,11 +227,62 @@ static void xbox_memory_init(PCMachineState *pcms,
     xbox_flash_init(rom_memory);
 }
 
+uint8_t* load_eeprom(void)
+{
+    char *filename;
+    int fd;
+    int rc;
+    int eeprom_file_size;
+    const int eeprom_size = 256;
+
+    uint8_t *eeprom_data = g_malloc(eeprom_size);
+
+    const char *eeprom_file = object_property_get_str(qdev_get_machine(),
+                                                      "eeprom", NULL);
+    if ((eeprom_file != NULL) && (strlen(eeprom_file) > 0)){
```

Comment:
Please undo this change - the original code was more similar to existing code (the `bootrom_file`).
If `*eeprom_file` is the standard way of doing it, we should keep doing so (or change *all* instances at once - which doesn't fit into this PR).